x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/auto_follow_pattern_list.js

```diff
@@ -202,6 +203,7 @@ export const AutoFollowPatternList = injectI18n(
               {...routing.getRouterLinkProps('/auto_follow_patterns/add')}
               fill
               iconType="plusInCircle"
+              data-test-subj="ccrCreateAutoFollowPatternButton"
```

Comment:
I think we should strive for unique selectors, even if our UI shouldn't show them both at the same time. This way if there is some kind of bug, and the _wrong_ button is rendered, the test will fail when it can't find the desired button, which will be easier to debug.

In this case, I'd suggest changing `ccrCreateAutoFollowPatternButton` to something like `ccrAutoFollowPatternEmptyPromptCreateButton`.